src/Modules/AuthModule/components/ForgetPass/ForgetPass.tsx

```diff
@@ -1,5 +1,95 @@
-import React from "react";
+import { useForm } from "react-hook-form";
+import logo from "../../../../assets/images/PMS 3.png";
+import { useNavigate } from "react-router-dom";
+import { useState } from "react";
+import axios from "axios";
+import { useToast } from "../../../../Context/ToastContext";
+import { FormData } from "../../../../interfaces/Auth";
 
 export default function ForgetPass() {
-  return <>ForgetPass</>;
+	const [spinner, setSpinner] = useState<boolean>(false);
+	const { showSuccessToast, showErrorToast } = useToast();
+
+	const {
+		register,
+		handleSubmit,
+		formState: { errors },
+	} = useForm<FormData>();
+	const navigate = useNavigate();
+
+	const onSubmit = async (data: FormData) => {
+		setSpinner(true);
+
+		try {
+			const response = await axios.post(
+				`https://upskilling-egypt.com:3003/api/v1/Users/Reset/Request`,
+				data
+			);
+			showSuccessToast(
+				"Your request is being processed, please check your email"
```

Comment:
We need to provide the response coming from the backend, and if the is no message, we should provide appropriate default message.